.npmrc

```diff
@@ -0,0 +1 @@
+@cultureamp:registry=https://npm.pkg.github.com/
```

Comment:
This lets yarn know which registry to use for `@cultureamp`-scoped packages (i.e. the GitHub registry).